### PR TITLE
Added doc regarding blobfuse2, changed tekton api version in examples

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: public-site/docs/src/references/reference-radix-config/index.md [hunk 1]
```diff
@@ -444,25 +444,31 @@ spec:
...
 #### `volumeMounts` settings
 
 - `name` - the name of the volume. Unique within `volumeMounts` list of a component
-- `type` - type of storage. Supported types:
-  - `azure-blob` - mount a container from blob in [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview). Uses [CSI Azure blob storage driver](https://github.com/kubernetes-sigs/blob-csi-driver). Replaces obsolete type `blob` for Flex Volume obsolete driver.
+- `path` - the folder inside the running container, where the external storage is mounted.
+- `blobfuse2` - mount a container from blob in [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview). Uses [CSI Azure blob storage driver](https://github.com/kubernetes-sigs/blob-csi-driver). Replaces types `blob` and `azure-blob` for obsolete drivers.
 
-_Applicable for type: `azure-blob`_
+_Options off `blobfuse2`_
```
Comment (emirgens) [inline public-site/docs/src/references/reference-radix-config/index.md:463]: Do you mean options _for_

**Review by emirgens — APPROVED** [review]
see my comments (2)

File: public-site/docs/src/references/reference-radix-config/index.md (hunk 1 above)
Comment (satr) [reply public-site/docs/src/references/reference-radix-config/index.md]: fixed

File: public-site/docs/src/guides/volume-mounts/index.md
[the remark this answers is not in the document]
Comment (satr) [reply public-site/docs/src/guides/volume-mounts/index.md:1]: fixed

**Pull request merged**